### ISSUE TITLE
Logo missing in printed version of specs

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/tr-design#242]
When printing, the W3C logo becomes: 

<img width="720" alt="a white square, with a blue border" src="https://user-images.githubusercontent.com/870154/114797481-9e570f80-9dd6-11eb-9eac-bf345e508fd2.png">

**Comment by deniak** [comment]
@fantasai, is there a way to update the svg so we don't rely on a blue background?

**Comment by marcoscaceres** [comment]
Argh, my bad. I hadn't checked the "print backgrounds" option (which, **I think**, is off by default).  

<img width="304" alt="Screen Shot 2021-04-19 at 4 29 32 pm" src="https://user-images.githubusercontent.com/870154/115191183-6defd800-a12c-11eb-82a5-3ceeb91c1ccb.png">

I'd be ok to close this, but maybe the W3C logo should be a real image instead of a background image.

**Comment by fantasai** [comment]
It is a real image. It's just laid over a colored background.

**Comment by marcoscaceres** [comment]
Ah, got it. I wonder if we should change that to a `<picture>` element with a `<source media=>` specifically for print?

**Comment by Malvoz** [comment]
Stumbling upon this issue I was reminded of https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/294 (in which I incorrectly had assumed one alternative to solve the issue would be to move from background-images to `<img>`).

A `<source media="">` for print could work, where the source points to an image that is an "inverted" version (no/white background-color and colored text). But how would specs that use their own logos deal with that? For example: https://www.w3.org/TR/responsible-use-spatial/ (although that OGC logo happens to have a white background, so not an issue in that case). There'd have to be a way to supply 2 versions of a logo _and_ spec editors would have to be made aware of this potential problem.

Another option which I think is a more straight forward solution is to force printers to print the logos using [`-webkit-print-color-adjust: exact`](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-print-color-adjust) and [`color-adjust: exact`](https://developer.mozilla.org/en-US/docs/Web/CSS/color-adjust), e.g.:

```css
@media print {
  .head .logo {
    -webkit-print-color-adjust: exact;
                  color-adjust: exact;
  }
}
```
which results in:

![force-logo-print](https://user-images.githubusercontent.com/26493779/124438520-92953c80-dd78-11eb-9bfb-20f7b73c13c7.png)

**Comment by fantasai** [comment]
I think this was actualy fixed in 2016, see e8e22b4b1b8958db604e485fd8af50d38fbe1311 it looks like @deniak didn't copy it over to the server?

@deniak can you actually sync the entire folder instead of doing one-off file copies? Maybe that would be a better idea?

**Comment by deniak** [comment]
> I think this was actualy fixed in 2016, see [e8e22b4](https://github.com/w3c/tr-design/commit/e8e22b4b1b8958db604e485fd8af50d38fbe1311) it looks like @deniak didn't copy it over to the server?
> 
> @deniak can you actually sync the entire folder instead of doing one-off file copies? Maybe that would be a better idea?

It's now sync'ed. In production, the JS and stylesheets/logos aren't in the same folder but I wrote a small script for next time to make sure *all* the files are properly sync'ed

**Comment by marcoscaceres** [comment]
Confirming it's fixed! Thanks @fantasai and @deniak!

Closing.
